ts/Drawer/index.tsx

```diff
@@ -141,10 +131,7 @@ export class PersistentDrawer extends MaterialComponent<
       this.MDComponent.unlisten('MDCPersistentDrawer:open', this.open);
       this.MDComponent.destroy();
     }
-  }
-
-  public componentWillUpdate(nextProps) {
-    toggleDrawer(this.props, nextProps, this.MDComponent);
```

Comment:
delete `toggleDrawer` function may be?

Reply:
It's used by one of the other drawers  which uses `this.MDComponent.tabbar` instead of `this.MDComponent`, so that I can't replace it.

Reply:
@cromefire While I reviewed the code I saw that `this.MDComponent.tabbar` is used in `/ts/Tabs/index.tsx` and the method you don't want to remove is actually called `setActiveTabIndex` and is not the method `toggleDrawer`. So you should be able to remove `toggleDrawer` without any issues.

Reply:
Check that again, but I think is was used

Reply:
Sorry wrong function